website-custom-views/gatsby-config.mjs

```diff
@@ -1,5 +1,7 @@
 // eslint-disable-next-line import/extensions
 import colorPresets from '@commercetools-docs/gatsby-theme-docs/color-presets/index.mjs';
+const { isProd } = process.env;
```

Comment:
We don't have this defined here in this repo.

Reply:
That's fine, I'll remove it and we can stick to the prod url
]